core/src/view/view.cpp

```diff
@@ -281,21 +281,18 @@ void View::updateMatrices() {
             break;
     }
 
+    m_viewProj = m_proj * m_view;
+    m_invViewProj = glm::inverse(m_viewProj);
+
+    m_normalMatrix = glm::transpose(glm::inverse(glm::mat3(m_view)));
```

Comment:
Yep, normals are good with the axis contribution, something else is wrong in the pipeline.